addons/website_event_meet/models/event_type.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class EventType(models.Model):
+    _inherit = "event.type"
+
+    website_community = fields.Boolean("Website Community", help="Display community tab on website")
```

Comment:
Well, not a big fan of this naming.. community is quite vague..
Maybe 'is_meet_activated' ? But.. meh'.. 
help could be more precise : "Activate meeting rooms on the event and display community tab on website"
What do you think ?

Reply:
@dbeguin All customization option in the event sub-module start with `website_`, so maybe `website_meeting_room` ? (to be coherent with `website_track` and `website_track_proposal`)